libpromises/verify_reports.c

```diff
@@ -92,7 +92,7 @@ PromiseResult VerifyReportPromise(EvalContext *ctx, const Promise *pp)
 
     if (a.transaction.action == cfa_warn)
     {
-        cfPS(ctx, LOG_LEVEL_VERBOSE, PROMISE_RESULT_WARN, pp, a, "Need to repair reports promise: %s", pp->promiser);
+        cfPS(ctx, LOG_LEVEL_WARNING, PROMISE_RESULT_WARN, pp, a, "Need to repair reports promise: %s", pp->promiser);
```

Comment:
Dunno if it really makes sense to print a warning on a reports promise. For consistency, it would make sense, but from a usefulness point of view, not really.